fdbrpc/ActorFuzz.actor.cpp

```diff
@@ -94,7 +94,6 @@ ACTOR Future<int> actorFuzz0(FutureStream<int> inputStream, PromiseStream<int> o
 }
 
 ACTOR Future<int> actorFuzz1(FutureStream<int> inputStream, PromiseStream<int> outputStream, Future<Void> error) {
-	state int ifstate = 0;
```

Comment:
Not sure if we really care for now, but these lines are generated. If we ever need to regenerate them (we have occasionally), they will show back up.